sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/LiteralsShouldNotBePassedAsLocalizedParameters.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.ComponentModel;
+
+namespace Tests.Diagnostics
+{
+    class Program

```

Comment:
Ok was not sure about the behavior. I wanted a test with resx because I dislike the suggestion of the instance of resource manager with the `GetString` method, why use a string key when you can have something strongly typed like a property?